Validation/HGCalValidation/test/python/protoHGCalSimWatcher_cfg.py

```diff
@@ -26,62 +26,27 @@
-    process.load('Configuration.Geometry.GeometryExtended2026D49Reco_cff')
-    fileCheck = 'testHGCalSimWatcherV11.root'
-    runMode = 0
-elif (options.geometry == "D68"):
-    from Configuration.Eras.Era_Phase2C12_cff import Phase2C12
-    process = cms.Process('PROD',Phase2C12)
-    process.load('Configuration.Geometry.GeometryExtended2026D68_cff')
-    process.load('Configuration.Geometry.GeometryExtended2026D68Reco_cff')
-    fileCheck = 'testHGCalSimWatcherV12.root'
-    runMode = 0
-elif (options.geometry == "D83"):
+if (options.geometry == "D88"):
     from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
     process = cms.Process('PROD',Phase2C11M9)
-    process.load('Configuration.Geometry.GeometryExtended2026D83_cff')
-    process.load('Configuration.Geometry.GeometryExtended2026D83Reco_cff')
-    fileCheck = 'testHGCalSimWatcherV15.root'
-    runMode = 1
-elif (options.geometry == "D84"):
-    from Configuration.Eras.Era_Phase2C11_cff import Phase2C11
-    process = cms.Process('PROD',Phase2C11)
-    process.load('Configuration.Geometry.GeometryExtended2026D84_cff')
-    process.load('Configuration.Geometry.GeometryExtended2026D84Reco_cff')
-    fileCheck = 'testHGCalSimWatcherV13.root'
-    runMode = 0
-elif (options.geometry == "D88"):
-    from Configuration.Eras.Era_Phase2C11_cff import Phase2C11
-    process = cms.Process('PROD',Phase2C11)
-    process.load('Configuration.Geometry.GeometryExtended2026D88_cff')
     process.load('Configuration.Geometry.GeometryExtended2026D88Reco_cff')
     fileCheck = 'testHGCalSimWatcherV16.root'
     runMode = 1
 elif (options.geometry == "D92"):
-    from Configuration.Eras.Era_Phase2C11_cff import Phase2C11
-    process = cms.Process('PROD',Phase2C11)
-    process.load('Configuration.Geometry.GeometryExtended2026D92_cff')
+    from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
+    process = cms.Process('PROD',Phase2C11M9)
     process.load('Configuration.Geometry.GeometryExtended2026D92Reco_cff')
     fileCheck = 'testHGCalSimWatcherV17.root'
     runMode = 1
-elif (options.geometry == "D93"):
-    from Configuration.Eras.Era_Phase2C11_cff import Phase2C11
-    process = cms.Process('PROD',Phase2C11)
-    process.load('Configuration.Geometry.GeometryExtended2026D93_cff')
-    process.load('Configuration.Geometry.GeometryExtended2026D93Reco_cff')
-    fileCheck = 'testHGCalSimWatcherV17N.root'
-    runMode = 1
 else:
     from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
     process = cms.Process('PROD',Phase2C11M9)
-    process.load('Configuration.Geometry.GeometryExtended2026D77_cff')
-    process.load('Configuration.Geometry.GeometryExtended2026D77Reco_cff')
-    fileCheck = 'testHGCalSimWatcherV14.root'
-    runMode = 0
+    process.load('Configuration.Geometry.GeometryExtended2026D93Reco_cff')
+    fileCheck = 'testHGCalSimWatcherV17N.root'
+    runMode = 1
+
+print("Output file: ", fileCheck)
+print("Run Mode:    ", runMode)
 
```

Comment:
The same comment applies here as well. There's here the complication of handling correctly the `fileCheck` variable. Maybe the previous naming convention is be a better choice that could be applied here as well?

`runMode` is always set to 1: is it really needed?